### ISSUE TITLE
bbPress: Duplicated email content

### ISSUE THREAD

**Issue by dd32** [Automattic/gutenberg-everywhere#115]
There's reports that the bbPress notification emails are duplicating the content in emails.

~It seems like to duplicate it needs to be composed without the Editor active.~

See https://wordpress.slack.com/archives/C02RQC6RW/p1671207401967239

<img width="994" alt="Screen Shot 2022-12-17 at 10 33 00 am" src="https://user-images.githubusercontent.com/767313/208213228-8a5785c5-d614-468e-89d3-60a484ed7abc.png">

**Comment by Otto42** [comment]
> It seems like to duplicate it needs to be composed without the Editor active

Don't cross that off yet. It was a short email, but def duplicated without the block editor being used.

https://wordpress.org/support/topic/gutenberg-on-forums/#post-16296548

https://ibb.co/LQQtMN6

**Comment by johngodley** [comment]
I wasn't able to reproduce it locally, but in #117 I've added a check for blocks and if it's plain content then no modifications are made. I think this should fix the problem.

Based on the above screenshot it looks like the email format is slightly different and I think that might have thrown things. I've also updated the block-handling email code so it can handle these differences.

I've merged that fix in trunk here. Let me know if it helps and I'll make a proper release of 1.13.3
